infra/messaging/migrations/20190625030841-add-content-hash.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Message', 'contentHash', {
```

Comment:
Isn't the table name msg_message rather than Message ?

Reply:
I haven't fully tested this out.  Was using the name in the model.  You're probably right, and I'll make sure to test the migration before bringing this PR out of draft.